packages/core/src/components/Menu/MenuItem/components/BaseMenuItem/BaseMenuItem.types.ts

```diff
@@ -0,0 +1,31 @@
+import { MenuItemProps } from "../../MenuItem";
+import { MenuChild } from "../../../Menu/MenuConstants";
+import React from "react";
+import { VibeComponentProps } from "../../../../../types";
+
+export type LossMenuItemProps = Omit<
```

Comment:
What's this for?

Reply:
to not cause breaking change, I'm taking the MenuItem props, but just what I need, and to override those who are conflicting